source/arch/begin_run_event.json

```diff
@@ -0,0 +1,50 @@
+{
+    "properties": {
+        "beamline_config": {
+            "type": ["object", "string"],
+            "description": "Meta-data regrading how the beamline is configured.  Maybe UID into another collection"
+        },
+        "project": {
+            "type": "string",
```

Comment:
'type' describes the allowed datatypes of the project field. There is no `project.type` field, just a `project` field that must be a string.